feature-libs/product/configurators/common/assets/translations/translations.ts

```diff
@@ -0,0 +1,21 @@
+import { TranslationChunksConfig, TranslationResources } from '@spartacus/core';
+import { en } from './en/index';
+
+export const configuratorTranslations: TranslationResources = {
+  en,
+};
+
+// expose all translation chunk mapping for configurator sub features
+export const configuratorTranslationChunksConfig: TranslationChunksConfig = {
+  configurator: [
+    'configurator.header',
+    'configurator.tabBar',
+    'configurator.attribute',
+    'configurator.button',
+    'configurator.priceSummary',
+    'configurator.addToCart',
+    'configurator.overviewForm',
+    'configurator.group',
+    'configurator.conflict',
```

Comment:
I'm not sure we need those translation chunks. I believe it suffices:
````
configurator: [
   'configurator'
]
```
Or we can even drop whole configurator chunks mapping, becuase 1:1 names are mapped implicitly by default. 
For more see docs https://sap.github.io/spartacus-docs/i18n/#config-of-chunks-and-namespaces-mapping
Those mappings are needed only for lazy-loadable translation assets.

Reply:
Yes, I see you are right...removed this